comp/api/authtoken/component.go

```diff
@@ -9,16 +9,21 @@
 package authtoken
 
 import (
+	"crypto/tls"
+
```

Comment:
I think the blank here should be removed, which let's the formatter keep the lines tidy.

Reply:
It seems that `go fmt` format it this way.